frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -1085,6 +1085,14 @@ export const widgets = [
           schema: { type: 'union', schemas: [{ type: 'string' }, { type: 'number' }] },
         },
       },
+      backgroundColor: {
+        type: 'color',
+        displayName: 'Background Color',
+        validation: {
+          schema: { type: 'string' },
+        },
+        value: '#fff',
```

Comment:
Not here, in the definitions schema.
Ref: https://github.com/ToolJet/ToolJet/pull/4356/files#diff-05ef203383ea521c66bce82f07f8b921e28441a4818ab30af45ed08d4e4d5b6bR510